agent/ansible/pbench/agent/roles/pbench_agent_install/tasks/main.yml

```diff
@@ -1,11 +1,11 @@
 ---
-- import_role:
+- name: import clean_yum_cache
```

Comment:
While the specific action is "import_role", doesn't this actually execute the role?  So we might want to consider a name that indicates an action being taken from the admins point of view.